### ISSUE TITLE
Get rid of Unknown build commit email popup

### ISSUE THREAD

**Issue by KevinGrandon** [buildkite/feedback#394]
This popup is pretty infuriating as it constantly pops up and blocks build output. There's also no option to turn it off completely.

![image](https://user-images.githubusercontent.com/122602/40135623-9f70a288-58fa-11e8-923f-617856a0bd32.png)

Some options:
* Delete the code, turning it off for good.
* Don't show it again if I close it a single time.
* Have a "don't ask again" checkbox within the popup.

**Comment by petemounce** [comment]
I have two emails in GitHub account and same two in buildkite. I get this popup for every user pushing to repository that buildkite builds, when that person does not have a buildkite account. I'm a GitHub org admin, and a buildkite org admin.

If there were an option to turn it off, I'd use it.

**Comment by keithpitt** [comment]
Yeah, as @ticky said - if you hit the "Not Mine" button, it should never show you again for that email. If you do keep seeing it, it's a bug and we'll fix it! I think clicking outside the box will just hide it, and we'll prompt you again next time you reload the page.
